pom.xml

```diff
@@ -280,8 +284,10 @@
         <reflections.version>0.9.10</reflections.version>
         <breeze.version>1.0</breeze.version>
 
-        <javacpp.parser.skip>false</javacpp.parser.skip>     <!-- To skip header file parsing phase: -Djavacpp.parser.skip=true  -->
-        <javacpp.compiler.skip>false</javacpp.compiler.skip> <!-- To skip native compilation phase: -Djavacpp.compiler.skip=true -->
+        <javacpp.parser.skip>false
+        </javacpp.parser.skip>     <!-- To skip header file parsing phase: -Djavacpp.parser.skip=true  -->
+        <javacpp.compiler.skip>false
+        </javacpp.compiler.skip> <!-- To skip native compilation phase: -Djavacpp.compiler.skip=true -->
```

Comment:
Is this actually equivalent to the variant without the new line?

Reply:
Hm, it has been done by Idea formatter because of long line... but we can go with new line too...